main.js

```diff
@@ -29,8 +29,8 @@ exec(`git for-each-ref --sort=-creatordate --count 1 --format="%(refname:short)"
 
         console.log('\x1b[32m%s\x1b[0m', `Found tag: ${tag}`);
         console.log('\x1b[32m%s\x1b[0m', `Found timestamp: ${timestamp}`);
-        fs.appendFileSync(process.env.GITHUB_OUTPUT, `tag=${tag}\r\n`);
-        fs.appendFileSync(process.env.GITHUB_OUTPUT, `timestamp=${timestamp}\r\n`);
+        fs.appendFileSync(process.env.GITHUB_OUTPUT, `tag=${tag}`);
+        fs.appendFileSync(process.env.GITHUB_OUTPUT, `timestamp=${timestamp}`);
```

Comment:
This should do the trick: 

```suggestion
        fs.appendFileSync(process.env.GITHUB_OUTPUT, `tag=${tag}\n`);
        fs.appendFileSync(process.env.GITHUB_OUTPUT, `timestamp=${timestamp}\n`);
```

Reply:
Updated.